phasta/adaptLvlSet_loop.cc

```diff
@@ -36,9 +36,13 @@ namespace {
 }
 
 int main(int argc, char** argv) {
+#ifndef SCOREC_NO_MPI
   MPI_Init(&argc, &argv);
+#else
+  (void) argc, (void) argv;
+#endif
```

Comment:
Is there a way we can get rid of the `#ifdef` from all the user code? One potentially bad idea: 
define a `PCU_Initialize` and `PCU_finalize` functions and put this code inside those. Until we have proper MPI sessions support, I'm not sure if there is much better that we can do.